CODE_OF_CONDUCT.md

```diff
@@ -1,23 +1,123 @@
-## Code of Conduct
+# Contributor Covenant Code of Conduct
 
-This code of conduct applies to all spaces provided by the Valkey project including in code, documentation, issue trackers, mailing lists, chat channels, wikis, blogs, social media and any other communication channels used by the project.
+## Our Pledge
 
-**Our open source communities endeavor to:**
+We as members, contributors, and leaders pledge to make participation in our
```

Comment:
Now is some type for stylistic questions I guess. We adopted one sentence per line here: https://github.com/valkey-io/valkey-doc?tab=readme-ov-file#styling-guidelines. Do you have thoughts about this?

Reply:
@madolson I'm fine with that with one caveat: there are a few corner cases where it's not possible to achieve the desired rendered HTML with line breaks after every period (e.g. when using liquid templating to vary a sentence or in front-matter).

So, as long as we don't enforce this with some merge blocking linter, I'm good with it.

Reply:
I see a "start each sentence on a new line" but nothing that says the sentence has to end on that line in https://github.com/valkey-io/valkey-doc?tab=readme-ov-file#styling-guidelines

One sentence per line can conflict with this other entry in the guidelines: "Please avoid writing lines that are too long, ..."

Reply:
Yeah, I'm fully aligned with better defined style guides. The "one sentence per line" is something I've come to like specifically because of how it makes it clear when sentence structure is changing, but understand it's not always possible.